src/config/consts.js

```diff
@@ -57,6 +57,10 @@ export const consts = {
     WEATHER: 3600000
   },
 
+  IMAGE_TYPE_REGEX: /\.(gif|jpe?g|tiff?|png|webp|bmp)$/i,
+  JPG_TYPE_REGEX: /\.(([jJ][pP][eE]?[gG]))$/,
+  PDF_TYPE_REGEX: /\.(([pP][dD][fF]))$/,
```

Comment:
we talked about it. please find a more appropriate solution :)

Reply:
[096984c](https://github.com/ikuseiGmbH/smart-village-app-app/pull/536/commits/096984c64d992d9da3140016b63ccff108960dcf)